### DISCUSSION TITLE
Woche 24.12 - 7.01.24

### DISCUSSION THREAD

**Discussion by carolina-cza** [carolina-cza/Tecna#5] [Weekly Blog]
moin, moin. 
Über die Ferien hab ich bisschen mit turtle und thinker gearbeitet. Auf die Ideen hat mich ein Freund gebracht, deswegen sind die bisschen random aber trotzdem ganz cool. Unter anderem auf diesen code [daily_stuff/tkinter/moreWindowsSmoke.py](url),background story dazu war das wir drüber geredet haben oder eher disskutiert das er endlich mit dem Rauchen aufhört, aber über zeugt hab ich ihn leider nicht davon.
Paar weitere Codes waren unteranderem an Weihnachten dieser code [daily_stuff/Turtle/ChristmasTree.py](url) und am ende der Ferien noch den Code, der eigentlich total random war aber ich den trotzdem toll fand [.](https://github.com/carolina-cza/Tecna/blob/codespace-super-space-yodel-x55ppqjx97jqfp699/daily_stuff/Turtle/birthdaycard.py) und ja das war eigentlich gröbstens was ich in den Ferien gemacht hab ^^

**Comment by JonNoe** [comment]
Finde deine Ideen mit der turtle und thinker Sachen darzustellen, die zur Jahreszeit oder deinem Alltag oder so passen toll. Besonders der Geburtstagkuchen hat  mir gefallen.

Mir ist aufgefallen, dass deine Verlinkungen nicht funktionieren. Ich hatte das selbe Problem, bei mir hats geholfen einfach den Link aus der Suchleiste reinzukopieren anstatt direkt von deinem Github Upload.

**Comment by dhbwJK23** [comment]
Interessant deine Diskussion mit dem Coden zu verbinden, vielleicht überzeugst du ihn ja noch.
Solange es dir beim Coden hilft hat es ja zumindest einen positiven Aspekt.
